trace/trace_opentelemetry/main.go

```diff
@@ -0,0 +1,39 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START trace_opentelemetry]
+
+func main() {
+
+    // Create exporter
+    exporter, err := stackdriver.NewExporter(
+        stackdriver.WithProjectID("PROJECT-ID"),
+        )
```

Comment:
Please read the project ID on a separate line and assign it to a `projectID` variable.

```
projectID := os.Getenv("GOOGLE_CLOUD_PROJECT")
```

Alternatively, you could make a `flag.String` so it can be passed as an argument. But, the env var is more common.